app/src/lib/shell.ts

```diff
@@ -0,0 +1,155 @@
+/* tslint:disable:no-sync-functions */
+
+import * as ChildProcess from 'child_process'
+import * as os from 'os'
+
+const ENVIRONMENT_VARIABLES_TO_PRESERVE = new Set([ 'NODE_ENV', 'NODE_PATH' ])
```

Comment:
I've :fire:'d those special cases, we can come back to this later once we understand this better and why we shouldn't use the ones from the shell.